### ISSUE TITLE
Explanation of deep sort architecture

### ISSUE THREAD

**Issue by sandeshshrestha45** [ZQPei/deep_sort_pytorch#11]
Dear ZQPei, 
In the train chart, there are two loss charts named loss(left) and top1err(right). if the top1err represents the error after softmax, then what does the the "loss" chart represent?
Also, what should be the general output of the mAP value of toperror while running evaluation.py?

**Comment by ZQPei** [comment]
Hello!
Top1err is not the error after softmax. Softmax outputs the probability of each class.
loss = cross_entropy(prob, target)
top1err = correct_predicted / total
Loss is the value to optimize on training set, and top1err is the percentage of error predictions on val set.

**Comment by sandeshshrestha45** [comment]
Thank you ZQPei, 
In your model,  it seems that there are more layers than the original layer. Is it true?  If it is true,  how many players have you added?  Also what are the other changes you have made?

**Comment by ZQPei** [comment]
Yes, I changed the model a little bit by adding some conv layers. You can also customize your model in the same way.
With a deeper model, you will get a higher accuracy.
There are no other changes I think.
